google/datalab/bigquery/_table.py

```diff
@@ -636,46 +636,6 @@ def to_dataframe(self, start_row=0, max_rows=None):
     ordered_fields = [field.name for field in self.schema]
     return df[ordered_fields] if df is not None else pandas.DataFrame()
 
-  def to_file(self, destination, format='csv', csv_delimiter=',', csv_header=True):
```

Comment:
note this has the actual extract functionality that the `Query` object uses. if we don't want to expose it we should just make it private, but we can't remove it altogether.